test/e2e/app-dir/parallel-routes-and-interception/parallel-routes-and-interception.test.ts

```diff
@@ -165,7 +165,9 @@ createNextDescribe(
         await step5()
       })
 
-      it('should match parallel routes', async () => {
+      // FIXME: this parallel route test is broken, shouldn't only check if html containing those strings
+      // previous they're erroring so the html is empty but those strings are still in flight response.
+      it.skip('should match parallel routes', async () => {
```

Comment:
checked with @feedthejim those are broken as empty page on canary, but they're rendering properly 404 page after this fix. so we disable them now and follow up in another task NEXT-1440